app/build.gradle

```diff
@@ -48,21 +48,19 @@ ktlint {
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     //Android Support Libs
-    implementation 'com.android.support:appcompat-v7:27.1.1'
-    implementation 'com.android.support:design:27.1.1'
-    implementation 'com.android.support.constraint:constraint-layout:1.1.2'
+    implementation 'androidx.appcompat:appcompat:1.0.0-rc02'
+    implementation 'com.google.android.material:material:1.0.0-rc01'
+    implementation 'androidx.constraintlayout:constraintlayout:1.1.2'
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tumugin/VirtualChemLight/blob/ca04fa5a4be76dc14eebc0c643e5b311716df235/app/build.gradle#L53"></span>A newer version of androidx.constraintlayout:constraintlayout than 1.1.2 is available: 1.1.3</td>
    </tr>
  </tbody>
</table>